scout/server/blueprints/managed_variants/templates/managed_variants/managed_variants.html

```diff
@@ -154,8 +154,20 @@ <h5>
       <td><input type="number" class="form-control" name="end" id="end" value="{{ variant.end }}"></td>
       <td><input type="text" class="form-control" name="reference" id="reference" value="{{ variant.reference }}" required></td>
       <td><input type="text" class="form-control" name="alternative" id="alternative" value="{{ variant.alternative }}" required></td>
-      <td>{{ modify_form.category(class_="form-control", onchange="populateSubTypeSelect('modify_variant')") }}</td>
-      <td>{{ modify_form.sub_category(class_="form-control") }}</td>
+      <td>
+        <select name="category" id="category" class="form-control" onchange="populateSubTypeSelect('modify_variant_{{variant.variant_id}}')">
+          {% for ctg,ctg_display in modify_form.category.choices %}
+            <option value="{{ctg}}" {% if ctg == variant.category %} selected {% endif %}>{{ctg_display}}</option>
+          {% endfor %}
+        </select>
+      </td>
+      <td>
+        <select name="sub_category" id="sub_category" class="form-control">
+          {% for subctg,subctg_display in modify_form.sub_category.choices %}
```

Comment:
Ah, this needs a little extra treatment, as `populateSubTypeSelect` will not be called until a category is actively changed/selected. Basically we need to do what that routine does when prepopulating instead, OR perhaps even better, since it should be modified now anyway, in the `document.ready()`.